### DISCUSSION TITLE
Alguem me consegue ajudar a resolver este erro por favor?

### DISCUSSION THREAD

**Discussion by moreira1313** [elizandrodantas/bot-blaze-telegram#33] [General]
executo o bot e da tudo certo mas depois aparece me este erro e eu nao sei como resolver

![bot](https://user-images.githubusercontent.com/92185183/222189913-c5615f7c-1e6c-45e6-b7a2-59395d2c9d5d.png)

alguem me conseguiria ajudar por favor?

**Comment by DinhoNunesLC** [comment]
Seu bot precisa ser admin do canal/grupo ao qual vai receber os sinais.
